src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -31,7 +31,7 @@ function parseFakeFHIRId(id) {
 }
 
 function RequestedAppointmentDetailsPage({
-  appointment,
+  appointmentDetails,
```

Comment:
I think the changed in this file might also not be intentional

Reply:
Fixed.